src/components/pages/AllPhrases.js

```diff
@@ -24,7 +27,9 @@ export default function AllPhrases({
     <AllPhrasesWrapper>
       {noPhrases}
 
-      <PhrasesList role="list" aria-label="phrases">
+      {isLoading ? <ModalLoadingInfo loadingStatus={loadingStatus} /> : null}
```

Comment:
Maybe replace with:
`{isLoading && <ModalLoadingInfo loadingStatus={loadingStatus} />}`

Reply:
thx good point...I´ve changed it

Reply:
I´ve changed it back...because have a look to this link: [Use ternaries rather than && in JSX](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)